### DISCUSSION TITLE
Some public sites are actually private?

### DISCUSSION THREAD

**Discussion by rezad1393** [Purfview/IMDb-Scout-Mod#49] [Q&A] [answered]
login needed:
https://eliteunitedcrew.org/login.php?returnto=%2Fbrowse.php%3Fsearch%3D%25tt%25%26cat%3D0%26incldead%3D1%26btn%3DSearch

https://mygully.com/search.php?do=process&quicksearch=1&childforums=1&exactname=1&query=%tt%

https://deildu.net/login.php?returnto=%2Fbrowse.php%3Fsearch%3D%25tt%25%26cat%3D0%26Lysing%3D1

https://www.dnoid.pw/account_handler.php?returnpath=%2Ffiles%2F%3Fquery%3D%25tt%25&withq=1

https://forum.dirtywarez.com/search/search?keywords=%tt%

**Comment by Purfview** [comment, marked as answer]
"Login needed" doesn't mean that site is not public. If everyone can login = public.

**Reply by mastercho** [reply]
i think he mean site like https://eliteunitedcrew.org/ need invite, so all invite websites i guess they are private not public as you cant signup anytime

**Reply by Purfview** [reply]
Yeap, it needs invite. I'll move it to the private sites next patch. Btw, what's up with EUCv2 there, new redesign?

It happens that sites move from private to public and from public to private, if you notice any at wrong section then report it.

**Reply by rezad1393** [reply]
I actually meant private as in you need to register.
but I messed up the private site with private tracker.
I meant private website.
for example zooqle in not a private website but those websites I mentioned (that they may not be private tracker) are private website because they need id to login.

I got entangeled with this when I enabled the experimental no third party cookie and site isolation a long time ago to test and that cause the website that need cookie ( so username password needed) to fail on the imdb-scout script.

**Reply by Purfview** [reply]
> ...those websites I mentioned (that they may not be private tracker) are private website because they need id to login.

Sites on this script doesn't have any sorting for private/public trackers, only for private/public websites.
Like I said above - a public site can have login/cookies, that doesn't make it private. Like at every public library you need your ID, and that doesn't turn them into the private libraries.

**Reply by rezad1393** [reply]
>  Like I said above - a public site can have login/cookies, that doesn't make it private. Like at every public library you need your ID, and that doesn't turn them into the private libraries.

that word has different meaning for a buildings or institutions.
that word usually has less to do with registration and has to do with funding and access(entrance).

and that is a bad example even for your point of view because I can enter and browse in a public library and even use it with no id or registration (and before terror thingy ,even could use computer with no id, and still can in many places.  )

but if you want physical world equivalent then ok.
please separate private and public ones as you would a private and public park.

now back to cyber word, as I said the private and public thing is usually is apparent when using cookies.
so as I said when (not if) firefox restricts third party cookie even more then  the so called public one that you name wont work on a script run on a imdb.com page unless Mozilla changes the way it separates page ,program and domains.

so in that situation what I call public will still work and yours wont because it needs to send cookies along requests.

all this is semantics to make the script more readable as you done very correctly with German section (maybe add one for Chinese one or maybe just a separate section for non-english ones)

**Reply by Purfview** [reply]
> that word has different meaning for a buildings or institutions.

I don't see a difference.

> that word usually has less to do with registration and has to do with funding and access(entrance).

Exactly! Like I said, registration doesn't have anything to do with the "public" meaning!

> and that is a bad example even for your point of view because I can enter and browse in a public library and even use it with no id or registration

How this is "a bad example from my PoV" when I already said that registration doesn't have anything to do with a public sites.

> but if you want physical world equivalent then ok. please separate private and public ones as you would a private and public park.

That's how the sites are separated in the script already.

> now back to cyber word, as I said the private and public thing is usually is apparent when using cookies.

It's apparent to a registration, cookies has nothing to do with a private/public thing.

> so in that situation what I call public will still work and yours wont because it needs to send cookies along requests.

"In that situation" - in YOUR mixed-up correlated meanings of private/cookies/registration.
Looks like you are mixing private/public with login/no-login or with cookies/no-cookies.

**Private - not for everyone, public - for everyone.** There is no other meanings in it like the registrations, cookies or whatever else.

**Reply by Purfview** [reply]
> maybe add one for Chinese one or maybe just a separate section for non-english ones

If there is enough (~15) sites to split to a separate language/country section, then collect all the names of those sites and I would move them.
